homeassistant/components/wyoming/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""The Wyoming integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.discovery import async_load_platform
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Load Wyoming."""
+    stt_address = entry.options.get("stt_address")
```

Comment:
Why do we store two addresses? Shouldn’t it be one address and based on a capability query we decide to forward it to to STT or TTS?